projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.scss

```diff
@@ -1,21 +1,69 @@
-.container{
+.open-in-app-container {
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  align-items: center;
-}
-
-.mat-dialog-container{
-  padding: 5px;
-  border-radius: 36px;
-  background-color: var(--theme-blue-button-color);
+  place-content: center;
+  padding: 0;
+  border-radius: 8px;
+  background-color: var(--theme-primary-color);
   color: var(--theme-about-panel-background-color);
+  margin-top: 12px;
 }
 
 .open-in-app.mat-button {
   overflow-x: hidden;
+  font-size: 16px;
+  width: 100%;
+  padding: 0;
+  height: 48px;
 }
 
 .open-in-app.mat-button.cdk-program-focused .mat-button-focus-overlay {
   opacity: 0;
 }
+
+.open-in-app.mat-button.cdk-touch-focused {
```

Comment:
can you cleanup the class paths, too many useless classes for material components

Reply:
removed mat-button from here but in .open-in-app.mat-button I require it for overriding the padding property.

Reply:
also if I remove .mat-button from .cross-button.mat-button.cdk-program-focused .mat-button-focus-overlay this class then opacity propery will not override.

Reply:
removed .mat-button from here and from everwhere in this file